client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -244,6 +244,13 @@ const CheckoutThankYou = React.createClass( {
 			/* eslint-enable wpcalypso/jsx-classname-namespace */
 		}
 
+		// Rebrand Cities thanks page
+		if ( isRebrandCitiesSiteUrl( this.props.selectedSite.URL ) && PLAN_BUSINESS === this.props.selectedSite.plan.product_slug ) {
```

Comment:
I had to move the thank you page up here. As a result, the logic isn't working right. For testing purposes, I set this to `TRUE` so I could see what it looked like. 

`this.props.selectedSite.URL` passes the full wordpress.com url and it was returning false.